status-im/status-react
# feature #2696 - chat message after wallet send

fixes #2696

### Summary:

Sends a message in 1:1 chat every time user send a transaction (ETH or token) to a contact, whether the transaction was sent from chat or wallet. 

### Review notes (optional):

This PR removes `:execute-stored-command` since that was  temporary workaround implemented so that messages could be sent after the command was interrupted by a shortcut. Since we now have unified message handling for both chat and wallet transactions alike, that kludge is no longer needed.

### Testing notes (optional):

Transactions sent by pasting the address (or scanning the QR) will still not appear in chat history and will not display recipient name in transaction history (only address). This will be handled in a separate PR for issue #2884 

This PR removes the root cause of all of those "message with data from old transaction" bugs.

### Steps to test:
- Send transactions from chat (message should appear)
- Send transactions from wallet to contacts (message should appear)
- Send transactions from wallet to non-contacts or by address/qr (message should not appear)
- Check any of the conditions that triggered bugs where a message was sent with old transaction data or before it was updated.

status: ready
